webapp/src/jaas/reducers/store.ts

```diff
@@ -0,0 +1,14 @@
+import {applyMiddleware, createStore, Store} from 'redux';
+import thunk from 'redux-thunk';
+
+import {loadInitialState, JaaSState} from '../util';
+import combinedReducers from './';
+
+export function configureStore() : Store<JaaSState> {
+    const initialState = loadInitialState();
+    const store = createStore(combinedReducers, initialState, applyMiddleware(thunk));
+    return store;
+}
```

Comment:
@raghavaggarwal2308 What are your thoughts on this question?

Reply:
@mickmister For Jaas meeting we have used everything related to redux separate from that of mattermost. The main reason is that we don't want to load the entire mattermost redux state and use all of its functions just for the Jaas server meeting as this would be faster and will not reduce the performace.

Reply:
> we don't want to load the entire mattermost redux state and use all of its functions just for the Jaas server meeting as this would be faster and will not reduce the performance.

@Kshitij-Katiyar If you use selectors this shouldn't be an issue. Please follow the normal patterns of using the existing refux store. It makes debugging redux actions a lot easier is the whole app is using the same store.

Reply:
@mickmister Here is the user who is not a mattermost user who can join the Jaas Meeting as well. So I think loading the mattermost state and using all of its redux functions for a user who is not even a part of mattermost would definitely reduce the performance and I also need to see whether we can even load and use the mattermost redux functions for a non-mattermost user. That will be the main reason for using a separate redux store.

Reply:
@Kshitij-Katiyar Okay this seems pretty strange. Is this why there is a call to `ReactDOM.render` in this PR - for users that are not logged in? Something seems off about this and I'm wondering if there is another solution available. Let's rule out the performance concerns you're mentioning because I think it's negligible for the purpose of this discussion. cc @hmhealey in case you have any insights on this discussion

> I also need to see whether we can even load and use the mattermost redux functions for a non-mattermost user.

Please check if the PR works as intended for unauthenticated users, with the existing redux store

Reply:
@mickmister removed the above store and reducers here.